### ISSUE TITLE
setting Virtual Env from Team Server

### ISSUE THREAD

**Issue by mikelope** [Microsoft/PTVS#2361]
[Window Title]
Visual Studio - Python support

[Main Instruction]
An unexpected error occurred

[Content]
Please press Ctrl+C to copy the contents of this dialog and report this error to our issue tracker.

[Expanded Information]
```
System.InvalidOperationException: Operation cannot be started because the environment is not configured.
   at Microsoft.PythonTools.Interpreter.PipPackageManager.<AbortIfNotReady>d__24.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.Interpreter.PipPackageManager.<InstallAsync>d__35.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.EnvironmentsList.PipExtensionProvider.<InstallPackage>d__36.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.PythonTools.EnvironmentsList.PipExtension.<InstallPackage_Executed>d__14.MoveNext()
```

**Comment by zooba** [comment]
Could you open Tools, Python, Diagnostic Info and either post the contents as a [gist](https://gist.github.com) and link it here, or email as an attachment to ptvshelp@microsoft.com?

**Comment by zooba** [comment]
Dup of #2495, fixed in #2502
